airbyte-webapp/src/hooks/services/useConnectionHook.tsx

```diff
@@ -134,19 +134,21 @@ const useGetConnection = (connectionId: string, options?: { refetchInterval: num
 const useCreateConnection = () => {
   const service = useWebConnectionService();
   const queryClient = useQueryClient();
-
   const analyticsService = useAnalyticsService();
 
   return useMutation(
     async (conn: CreateConnectionProps) => {
       const { values, source, destination, sourceDefinition, destinationDefinition } = conn;
+
       const response = await service.create({
         sourceId: source?.sourceId,
         destinationId: destination?.destinationId,
         ...values,
         status: "active",
       });
 
+      const enabledStreams = values.syncCatalog.streams.filter((stream) => stream.config.selected === true).length;
```

Comment:
Nitpick: `selected` is already a boolean, no need to compare it to `true`

```suggestion
      const enabledStreams = values.syncCatalog.streams.filter((stream) => stream.config.selected).length;
```